account_global_discount/views/account_invoice_views.xml

```diff
@@ -0,0 +1,138 @@
+                    name="global_discount_ids"
+                    widget="many2many_tags"
+                    placeholder="Discounts..."
+                    readonly="state != 'draft'"
+                    groups="base_global_discount.group_global_discount"
+                />
+                <field
+                    string="Invoice Global Discounts"
+                    name="global_discount_ids_readonly"
+                    widget="many2many_tags"
+                    readonly="1"
+                    groups="!base_global_discount.group_global_discount"
+                />
+            </xpath>
+            <field name="tax_totals" position="before">
+                <field
+                    name="amount_untaxed_before_global_discounts"
+                    string="Untaxed Amount Before Disc."
+                    invisible="not global_discount_ids or not invoice_global_discount_ids"
+                    groups="base_global_discount.group_global_discount"
+                />
+                <field
+                    name="amount_untaxed_before_global_discounts"
+                    string="Untaxed Amount Before Disc."
+                    invisible="not global_discount_ids_readonly or not invoice_global_discount_ids"
+                    groups="!base_global_discount.group_global_discount"
+                />
+                <field
+                    name="amount_global_discount"
+                    string="Global Discounts"
+                    invisible="not global_discount_ids or not invoice_global_discount_ids"
+                    groups="base_global_discount.group_global_discount"
+                />
+                <field
+                    name="amount_global_discount"
+                    string="Global Discounts"
+                    invisible="not global_discount_ids_readonly or not invoice_global_discount_ids"
+                    groups="!base_global_discount.group_global_discount"
+                />
+            </field>
+            <field name="tax_totals" position="attributes">
+                <attribute
+                    name="class"
+                    add="oe_subtotal_footer_separator"
+                    separator=" "
+                />
+            </field>
+            <page id="other_tab" position="inside">
+                <separator string="Global Discounts" />
+                <field
+                    name="invoice_global_discount_ids"
+                    nolabel="1"
+                    readonly="1"
+                    invisible="not global_discount_ids"
+                    force_save="1"
+                    groups="base_global_discount.group_global_discount"
+                >
+                    <tree create="0" delete="0">
+                        <field name="name" />
+                        <field name="currency_id" invisible="1" />
+                        <field name="global_discount_id" invisible="1" />
+                        <field name="discount" invisible="1" />
```

Comment:
These fields should be applied with `column_invisible` as well

![imagen](https://github.com/user-attachments/assets/3b89d769-5cce-46f7-972a-72078c9f4ad9)